src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -299,10 +299,10 @@
     <DropUnsignedFile Include="$(OutDir)\Microsoft.MICore.dll.mdb" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.MIDebugEngine.dll.mdb" />
   </ItemGroup>
-  <ItemGroup Condition="'$(IsCoreCLR)' == 'false' OR '$(Lab)' == 'false'">
-    <!-- Include file for local VS development builds and all non-VS builds -->
+  <!--<ItemGroup Condition="'$(IsCoreCLR)' == 'false' OR '$(Lab)' == 'false'">
```

Comment:
Remove rather than just commenting out?

Reply:
@gregg-miskelly I commented out because my assumption is that we'll put them back when `mono` resolves their issue. I can remove it if you think it would be better?

Reply:
If you would rather leave them, that is fine. But in that case I would suggest adding a comment to indicate why you left it. Or you could just take it out since you can revert this change entirely at that point.

Reply:
I fixed this one so it still copies it out in non-lab regular builds. Otherwise I removed it.